### PR TITLE
INDY-902: Removal of deprecated code

### PR DESCRIPTION
**Pull request by ArtObr**

### CODE REVIEW

File: indy_client/test/cli/test_node_suspension.py [hunk 1]
```diff
@@ -1,35 +1,22 @@
...
+from plenum.common.constants import SERVICES, TARGET_NYM, DATA
 from plenum.common.txn_util import get_payload_data
 
-from plenum.test.pool_transactions.helper import sdk_add_new_nym, sdk_add_new_node, demote_node
-from indy_client.test.cli.helper import doSendNodeCmd
+from plenum.test.pool_transactions.helper import sdk_add_new_nym, sdk_add_new_node, demote_node, promote_node
 
 
-def testSuspendNode(be, do, trusteeCli, newNodeAdded):
+def testSuspendNode(looper, sdk_pool_handle, sdk_wallet_trustee, newNodeAdded):
```
Comment (ashcherbakov) [inline indy_client/test/cli/test_node_suspension.py]: Please move the test into indy-node folder

File: indy_client/test/cli/test_node.py [hunk 2]
```diff
@@ -1,173 +0,0 @@
...
-    ALIAS, BLS_KEY, BLS_KEY_PROOF
-from plenum.common.keygen_utils import init_bls_keys
-from plenum.common.util import randomString
-from plenum.test.cli.helper import exitFromCli
-from stp_core.network.port_dispenser import genHa
-
-from indy_client.test.cli.helper import doSendNodeCmd
-
-
-def test_add_new_node(newNodeAdded):
```
Comment (ashcherbakov) [inline indy_client/test/cli/test_node.py]: This tests need to be migrated to SDK and moved to indy-node folder

File: indy_client/test/cli/test_node_demotion.py [hunk 3]
```diff
@@ -1,51 +0,0 @@
-import pytest
-from plenum.common.signer_did import DidSigner
-from plenum.common.txn_util import get_payload_data
-from stp_core.crypto.util import randomSeed
-from indy_client.test.cli.helper import addAgent
-from plenum.common.constants import SERVICES, VALIDATOR, TARGET_NYM, DATA
-from indy_client.test.cli.constants import NODE_REQUEST_COMPLETED, NODE_REQUEST_FAILED
-
-
-def ensurePoolIsOperable(be, do, cli):
```
Comment (ashcherbakov) [inline indy_client/test/cli/test_node_demotion.py]: This tests need to be migrated to SDK and moved to indy-node folder

File: indy_client/test/cli/test_pool_upgrade.py [hunk 4]
```diff
@@ -1,137 +0,0 @@
...
-from indy_node.test import waits
-from stp_core.loop.eventually import eventually
-from plenum.common.constants import VERSION
-from indy_common.constants import ACTION, CANCEL, JUSTIFICATION
-from indy_node.test.upgrade.helper import checkUpgradeScheduled, \
-    checkNoUpgradeScheduled
-from indy_node.test.upgrade.conftest import validUpgrade, validUpgradeExpForceFalse, validUpgradeExpForceTrue
-
-
-def send_upgrade_cmd(do, expect, upgrade_data):
```
Comment (ashcherbakov) [inline indy_client/test/cli/test_pool_upgrade.py]: This tests need to be migrated to SDK and moved to indy-node folder

File: indy_client/test/cli/test_pool_upgrade_schedule.py [hunk 5]
```diff
@@ -1,27 +0,0 @@
-import dateutil
-import pytest
-from datetime import timedelta, datetime
-
-from indy_client.test.cli.test_pool_upgrade import poolUpgradeSubmitted
-from indy_client.test.cli.test_pool_upgrade import poolUpgradeScheduled
-from indy_node.test.upgrade.conftest import validUpgrade as _validUpgrade
-
-
-@pytest.fixture(scope='function')
```
Comment (ashcherbakov) [inline indy_client/test/cli/test_pool_upgrade_schedule.py]: This tests need to be migrated to SDK and moved to indy-node folder

File: indy_client/test/cli/test_pool_upgrade_validation.py [hunk 6]
```diff
@@ -1,44 +0,0 @@
...
-
-from indy_node.test.upgrade.conftest import validUpgrade
-from indy_client.test.cli.constants import INVALID_SYNTAX, ERROR
-from indy_node.test.upgrade.helper import loweredVersion
-from plenum.common.constants import VERSION
-from plenum.common.util import randomString
-from indy_common.constants import JUSTIFICATION, JUSTIFICATION_MAX_SIZE
-
-
-def testPoolUpgradeFailsIfVersionIsLowerThanCurrent(
```
Comment (ashcherbakov) [inline indy_client/test/cli/test_pool_upgrade_validation.py]: This tests need to be migrated to SDK and moved to indy-node folder

File: indy_client/test/cli/test_send_get_nym_validation.py [hunk 7]
```diff
@@ -1,153 +0,0 @@
...
-
-CURRENT_VERKEY_FOR_NYM = 'Current verkey for NYM {dest} is {verkey}'
-CURRENT_VERKEY_FOR_NYM_WITH_ROLE = 'Current verkey for NYM {dest} is ' \
-                                   '{verkey} with role {role}'
-CURRENT_VERKEY_IS_SAME_AS_IDENTIFIER = \
-    'Current verkey is same as identifier {dest}'
-NYM_NOT_FOUND = 'NYM {dest} not found'
-
-
-def testSendGetNymSucceedsForExistingUuidDest(
```
Comment (ashcherbakov) [inline indy_client/test/cli/test_send_get_nym_validation.py]: This tests need to be migrated to SDK and moved to indy-node folder

File: indy_client/test/cli/test_send_node_validation.py [hunk 8]
```diff
@@ -1,609 +0,0 @@
...
-from plenum.common.signer_simple import SimpleSigner
-from plenum.common.util import cryptonymToHex, randomString
-from indy_client.test.cli.conftest import newStewardCli as getNewStewardCli, \
-    newStewardVals as getNewStewardVals, newNodeVals as getNewNodeVals
-from indy_client.test.cli.constants import NODE_REQUEST_COMPLETED, NODE_REQUEST_FAILED, INVALID_SYNTAX
-from indy_client.test.cli.helper import addAgent
-
-NYM_ADDED = "Nym {remote} added"
-
-
```
Comment (ashcherbakov) [inline indy_client/test/cli/test_send_node_validation.py]: This tests need to be migrated to SDK and moved to indy-node folder

File: indy_client/test/cli/test_send_nym_validation.py [hunk 9]
```diff
@@ -1,541 +0,0 @@
...
-    hexToFriendly
-from indy_client.test.cli.constants import ERROR, INVALID_SYNTAX
-from indy_client.test.cli.helper import createUuidIdentifier, \
-    createHalfKeyIdentifierAndAbbrevVerkey, createUuidIdentifierAndFullVerkey, \
-    createCryptonym
-from indy_common.roles import Roles
-
-NYM_ADDED = 'Nym {dest} added'
-
-
```
Comment (ashcherbakov) [inline indy_client/test/cli/test_send_nym_validation.py]: This tests need to be migrated to SDK and moved to indy-node folder

File: indy_client/test/cli/test_node_suspension.py (hunk 1 above)
Comment (ArtObr) [reply indy_client/test/cli/test_node_suspension.py]: I thought to do this in the next PR. And this one mainly for deleting. Tell me, if we need to do that in this PR.

File: indy_client/test/cli/test_node.py (hunk 2 above)
Comment (ArtObr) [reply indy_client/test/cli/test_node.py]: We are having pretty same functionality in 
indy-plenum/plenum/test/pool_transactions/test_nodes_data_changed.py
Please, tell me, if we need to integrate this in node

File: indy_client/test/cli/test_node_demotion.py (hunk 3 above)
Comment (ArtObr) [reply indy_client/test/cli/test_node_demotion.py]: I think we have pretty same functionality in 
indy-plenum/plenum/test/pool_transactions/test_suspend_node.py
Please, tell me, if we need to integrate this in node

File: indy_client/test/cli/test_pool_upgrade.py (hunk 4 above)
Comment (ArtObr) [reply indy_client/test/cli/test_pool_upgrade.py]: I think we have every tests case duplicated in
indy-node/indy_node/test/upgrade
Please, tell me, if we need to integrate this in node

File: indy_client/test/cli/test_pool_upgrade_schedule.py (hunk 5 above)
Comment (ArtObr) [reply indy_client/test/cli/test_pool_upgrade_schedule.py]: I think we have this test case duplicated in
indy-node/indy_node/test/upgrade
Please, tell me, if we need to integrate this in node

File: indy_client/test/cli/test_send_nym_validation.py (hunk 9 above)
Comment (ArtObr) [reply indy_client/test/cli/test_send_nym_validation.py]: integrated

File: indy_client/test/cli/test_send_node_validation.py (hunk 8 above)
Comment (ArtObr) [reply indy_client/test/cli/test_send_node_validation.py]: integrated

File: indy_client/test/cli/test_send_get_nym_validation.py (hunk 7 above)
Comment (ArtObr) [reply indy_client/test/cli/test_send_get_nym_validation.py]: Restored. Will be integrated in future PRs

File: indy_client/test/cli/test_pool_upgrade_validation.py (hunk 6 above)
Comment (ArtObr) [reply indy_client/test/cli/test_pool_upgrade_validation.py]: Restored. Will be integrated in future PRs

**Pull request merged**